stories/breadcrumb.stories.js

```diff
@@ -29,7 +29,7 @@ storiesOf('Metadata/Breadcrumb', module)
           virtualHierarchicalMenu(),
 
           breadcrumb({
-            container: breadcrumb,
+            container: breadcrumbDiv,
```

Comment:
This currently breaks in the [prod deployment](http://instantsearchjs.netlify.com/stories?path=/story/metadata-breadcrumb--default).